collect_app/src/main/java/org/odk/collect/android/utilities/DialogUtils.java

```diff
@@ -154,13 +154,18 @@ public static Dialog createErrorDialog(@NonNull Activity activity, String errorM
         return alertDialog;
     }
 
-    public static <T extends DialogFragment> T showIfNotShowing(Class<T> dialogClass, FragmentManager fragmentManager) {
-        return showIfNotShowing(dialogClass, null, fragmentManager);
+    @Nullable
+    public static <T extends DialogFragment> T getDialog(Class<T> dialogClass, FragmentManager fragmentManager) {
```

Comment:
How about adding the notice from the PR here? ("as much as possible we should be using arguments (for static data the dialog needs), Dagger (for dependencies) or ViewModel (for non static data) to get things into fragments so as to avoid crashes or weirdness when they are recreated.")

Reply:
Ah I actually meant to do this but it slipped my mind. Will do!